charts/localstack/Chart.yaml

```diff
@@ -15,4 +15,8 @@ sources:
 icon: https://github.com/localstack/localstack/raw/master/localstack/dashboard/web/img/localstack.png
 maintainers:
 - name: Waldemar Hummer
-  email: waldemar.hummer@gmail.com
\ No newline at end of file
+  email: waldemar.hummer@gmail.com
+dependencies:
+  - name: common
+    version: 1.x.x
+    repository: https://charts.bitnami.com/bitnami
```

Comment:
Thanks for this PR @jhandguy ! 👍 Out of curiosity - is it required (or best practice) to check in the tgz files of the dependencies (as well as `Chart.lock`), or could it be removed (and added to a `.gitignore` file)? Thanks

Reply:
That's an excellent point actually, I was also wondering about that and let it be hoping to get some guidance. 
I've checked a few repos here and there and no one seems to be tracking those (i.e. https://github.com/artifacthub/hub/tree/master/charts/artifact-hub), I'll git ignore them right away! 👍

Reply:
Great, thanks. One more thing - can you please update the version to `0.1.2` in `Chart.yaml`. I believe this is required in order for the chart to get automatically released as part of our [Github Actions pipeline](https://github.com/localstack/helm-charts/actions). Thanks!